src/includes/feature-stage-beta-user-feedback.mdx

```diff
@@ -0,0 +1,5 @@
+<Alert level="info" title="A Wild User Feedback Appears!">
+
+We've released a new version of User Feedback that supports feedback without requiring an error event. It is currently in open beta and subject to change.
+
+</Alert>
```

Comment:
I think this would be better as a note!

```suggestion
<Note title="A Wild User Feedback Appears!">

Sentry released a new version of User Feedback that supports feedback without requiring an error event. It is currently in open beta and subject to change.

</Note>
```